src/components/ui/workscard/index.jsx

```diff
@@ -1,17 +1,29 @@
 import React from "react";
 
-import Button from "../Button";
-import IconButton from "../IconButton";
+import Button from "../button";
+import IconButton from "../iconbutton";
+import { attachClasses } from "../../../utils/index";
```

Comment:
1. This import statement should be after the classes import
2.  No need to add index in ../../../utils/index - only ../../../utils will work fine